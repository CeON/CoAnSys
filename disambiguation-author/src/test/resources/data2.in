c1	1	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2011)}]
c2	2	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2011)}]
c3	3	Kowalski	[EX_TITLE#{(An Empirical Study on the BAND)},EX_YEAR#{(2011)}]
c4	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Dlugi Tytul)},EX_YEAR#{(1996)}]
c5	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Dlugi Tytul 2)},EX_YEAR#{(1996)}]
c6	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Dlugi Tytul 3)},EX_YEAR#{(1996)}]
c7	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Dlugi Tytul 4)},EX_YEAR#{(1996)}]
c8	1	Nowak	[EX_TITLE#{(Kosmiczny Tytul)},EX_YEAR#{(1996)}]
c9	1	Nowak	[EX_TITLE#{(Kosmiczny Tytul)},EX_YEAR#{(1996)}]
c10	1	Nowak	[EX_TITLE#{(Kosmiczny Tytul)},EX_YEAR#{(1996)}]
c11	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Tytul 2)},EX_YEAR#{(1996)}]
c12	1	Nowak	[EX_TITLE#{(Samotnik)},EX_YEAR#{(2001)}]
