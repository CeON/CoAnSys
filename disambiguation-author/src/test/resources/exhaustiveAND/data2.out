(374259b3-d27b-3a3d-a809-f4af388d005f,{(d1),(d10),(d11),(d12),(d13),(d14),(d15),(d2),(d3),(d4),(d5),(d6),(d7),(d8),(d9)})
(3894d1ed-e73b-3abf-ac9f-20a1c9c73094,{(c3),(c4)})
(b285dd65-16bd-399a-a65c-d5a55d410f89,{(c1),(c2),(c5)})
(756787c3-d14e-3f1b-9645-e8914646010a,{(c6)})
(9b325bcc-06dd-3d7b-8c68-a219d1b59dff,{(c7),(c8),(c9)})
(6c536fa9-c0ec-390f-be81-66297ca9ba53,{(c10)})
(4d1fcec7-3688-36c7-919d-3335c026d122,{(c11)})
(fe68d6c1-ad26-3b37-bdc9-e0e211e92ae6,{(b1),(b2),(b3),(b4)})
(e071065d-01e7-3f4a-9596-90bd8c22d860,{(a1),(a2),(a3),(a4),(a5)})
